src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -49,21 +64,77 @@ Copyright (c) .NET Foundation. All rights reserved.
       $(AssignTargetPathsDependsOn)
     </AssignTargetPathsDependsOn>
 
-    <ResolveStaticWebAssetsInputsDependsOn>
+    <ResolveStaticWebAssetsInputsDependsOn Condition="$(NoBuild) != 'true'">
       _ResolveStaticWebAssetsProjectReferences;
       $(ResolveStaticWebAssetsInputsDependsOn)
     </ResolveStaticWebAssetsInputsDependsOn>
 
+    <GenerateStaticWebAssetsPackTargetsDependsOn>
+      _CreateStaticWebAssetsCustomPropsCacheFile;
+      $(GenerateStaticWebAssetsPackTargetsDependsOn)
+    </GenerateStaticWebAssetsPackTargetsDependsOn>
+
+    <TargetsForTfmSpecificContentInPackage>
+      GenerateStaticWebAssetsPackTargets;
+      $(TargetsForTfmSpecificContentInPackage)
+    </TargetsForTfmSpecificContentInPackage>
+
+    <PackDependsOn>
+      _RemoveWebRootContentFromPackaging;
+      $(PackDependsOn)
+    </PackDependsOn>
+
+    <!-- <BuildDependsOn>
+      GenerateStaticWebAssetsPackTargets;
+      $(BuildDependsOn)
+    </BuildDependsOn> -->
+
   </PropertyGroup>
 
   <PropertyGroup>
-    <_GeneratedStaticWebAssetsInputsCacheFile>$(IntermediateOutputPath)$(TargetName).StaticWebAssets.cache</_GeneratedStaticWebAssetsInputsCacheFile>
-    <_GeneratedStaticWebAssetsDevelopmentManifest>$(IntermediateOutputPath)$(TargetName).StaticWebAssets.xml</_GeneratedStaticWebAssetsDevelopmentManifest>
+    <_StaticWebAssetsIntermediateOutputPath>$(IntermediateOutputPath)staticwebassets\</_StaticWebAssetsIntermediateOutputPath>
+    
+    <!-- Development manifest generation -->
+    <_GeneratedStaticWebAssetsInputsCacheFile>$(_StaticWebAssetsIntermediateOutputPath)$(TargetName).StaticWebAssets.Manifest.cache</_GeneratedStaticWebAssetsInputsCacheFile>    
+    <_GeneratedStaticWebAssetsDevelopmentManifest>$(_StaticWebAssetsIntermediateOutputPath)$(TargetName).StaticWebAssets.xml</_GeneratedStaticWebAssetsDevelopmentManifest>
+    
+    <!-- Project packing generation -->
+    <_GeneratedStaticWebAssetsCustomPropsCacheFile>$(_StaticWebAssetsIntermediateOutputPath)$(TargetName).StaticWebAssets.Pack.cache</_GeneratedStaticWebAssetsCustomPropsCacheFile>
+
+    <!-- Temporary files -->
+    <_AutoGeneratedStaticWebAssetsPropsFile>$(_StaticWebAssetsIntermediateOutputPath)msbuild.$(PackageId).Microsoft.AspNetCore.StaticWebAssets.props</_AutoGeneratedStaticWebAssetsPropsFile>
```

Comment:
~~Auto~~Generated. Is there any other kind of Generated?

![maxresdefault](https://user-images.githubusercontent.com/174281/58274372-97890480-7d47-11e9-8b95-e61caeababa1.jpg)

Reply:
done